manticore/ethereum.py

```diff
@@ -173,6 +178,9 @@ def __build_source_map(self, bytecode, srcmap):
     def runtime_bytecode(self):
         # Removes metadata from the tail of bytecode
         end = None
+        if len(self._runtime_bytecode) < 44:
```

Comment:
Magic value of `44` - can we at least add a comment what does it mean?
(or eventually move to a meaningful named constant)